components/org.wso2.carbon.siddhi.editor.core/src/main/resources/web/js/main.js

```diff
@@ -141,6 +144,15 @@ define(['require', 'log', 'jquery', 'lodash', 'backbone', 'menu_bar', 'tool_bar'
                     this.workspaceManager.displayInitialTab();
                 },
 
+                runInitialGuide: function (){
+
+                    var isFreshUser = (this.browserStorage.get('guideFileNameIncrement') === null);
+                    if(isFreshUser) {
+                        this.workspaceManager.runGuide();
```

Comment:
Use the already created object to run the guide without using workspace

Reply:
Changed the object to the existing